src/mcp.jl

```diff
@@ -61,17 +61,14 @@ function get_initial_values_in_raw_index(m, mcp_data)
     return initial_values
 end
 
-function solveMCP(m::JuMP.Model; solver=:PATH, method=:trust_region, linear=false)
+function solveMCP!(m::JuMP.Model; solver=:PATH, method=:trust_region, kwargs...)
```

Comment:
Any reason why renaming? This just breaks things. I would still take the `linear=false` keyword, but just do nothing with it.

```julia
function solveMCP(m::JuMP.Model; solver=:PATH, method=:trust_region, linear = false, kwargs...)
    if linear
        @warn("The linear keyword argument has been deprecated. You don't need to set it anymore.")
    end
```

Reply:
Agreed. Thanks!